README.md

```diff
@@ -82,27 +77,47 @@ public:
 
         static void _register_methods() {
            register_method("method", &SimpleClass::method);
+	   
+	   /**
+	    * How to register exports like gdscript
+	    * export var _name = "SimpleClass"
+	    **/
+	   register_property((char *)"base/name", &SimpleClass::_name, Variant("SimpleClass"));
```

Comment:
The default value must have the se type as the member pointer but here it's Variant. If it is a String then you must put a String there